SourceryRuntime/Sources/Description.generated.swift

```diff
@@ -254,9 +254,9 @@ extension Type {
         string += "name = \(String(describing: self.name)), "
         string += "isGeneric = \(String(describing: self.isGeneric)), "
         string += "localName = \(String(describing: self.localName)), "
-        string += "variables = \(String(describing: self.variables)), "
-        string += "methods = \(String(describing: self.methods)), "
-        string += "subscripts = \(String(describing: self.subscripts)), "
+        string += "rawVariables = \(String(describing: self.rawVariables)), "
+        string += "rawMethods = \(String(describing: self.rawMethods)), "
+        string += "rawSubscripts = \(String(describing: self.rawSubscripts)), "
```

Comment:
Out of scope, but I bet a multiline string would help a little with compile time and a lot with formatting here